{"name": "Ошибка при неверном email", "status": "passed", "description": "Проверяем, что нельзя авторизоваться с несуществующим email", "steps": [{"name": "Отправка запроса на авторизацию с несуществующим email", "status": "passed", "steps": [{"name": "Авторизация пользователя", "status": "passed", "parameters": [{"name": "email", "value": "'nonemail@mail.ru'"}, {"name": "password", "value": "'1WRebIgunu'"}], "start": 1757439460232, "stop": 1757439460624}], "start": 1757439460232, "stop": 1757439460624}, {"name": "Проверяем, что при создании пользователя без обязательного поля код ответ 401 и success: false", "status": "passed", "start": 1757439460624, "stop": 1757439460624}, {"name": "Проверяем что сообщение об ошибке: email or password are incorrect", "status": "passed", "start": 1757439460624, "stop": 1757439460624}], "start": 1757439460232, "stop": 1757439460626, "uuid": "5943f789-91e5-423c-a760-358feeeebe2e", "historyId": "ebd94b3c8f038287042f1836657cd9fa", "testCaseId": "ebd94b3c8f038287042f1836657cd9fa", "fullName": "tests.test_auth_user.TestAuthUser#test_auth_user_invalid_email_failed", "labels": [{"name": "feature", "value": "API: Авторизация пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_user"}, {"name": "subSuite", "value": "TestAuthUser"}, {"name": "host", "value": "DESKTOP-RUV96V5"}, {"name": "thread", "value": "21508-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_user"}]}